pkg/noobaaaccount/noobaaaccount.go

```diff
@@ -374,6 +387,45 @@ func RunRegenerate(cmd *cobra.Command, args []string) {
 
 }
 
+// RunCredentials runs a CLI command
+func RunCredentials(cmd *cobra.Command, args []string) {
+	log := util.Logger()
+
+	if len(args) != 1 || args[0] == "" {
+		log.Fatalf(`❌ Missing expected arguments: <noobaa-account-name> %s`, cmd.UsageString())
+	}
+
+	name := args[0]
+
+	o := util.KubeObject(bundle.File_deploy_crds_noobaa_io_v1alpha1_noobaaaccount_cr_yaml)
+	noobaaAccount := o.(*nbv1.NooBaaAccount)
+	noobaaAccount.Name = name
+	noobaaAccount.Namespace = options.Namespace
+
+	var accessKeys nb.S3AccessKeys
+
+	accessKeys.AccessKey, _ = cmd.Flags().GetString("access_key")
+	accessKeys.SecretKey, _ = cmd.Flags().GetString("secret_key")
+
+	if accessKeys.AccessKey == "" || accessKeys.SecretKey == "" {
+		log.Fatalf(`❌ access_key and secret_key flags must be provided`)
+	}
```

Comment:
What did you think about adding a check here as well? We would have fewer RPC calls.
And also explain the credentials requirements.

Reply:
Yes, I am thinking of the same. I will add here as well. Thanks